src/telemetry/events.ts

```diff
@@ -20,6 +20,7 @@ export enum UserEvent {
   ViewSearchAction = "View Search Action",
   SchemaAction = "Schema Action",
   FlinkStatementAction = "Flink Statement Action",
+  FlinkStatementViewStatistics = "Flink Statement View Statistics",
```

Comment:
My only concern is this will turn into a totally separate table of data (which can be good if we really want it separated from the main user-actions), but may make it harder to analyze later if we want it incorporated into existing events